### ISSUE TITLE
com.alibaba.nacos.api.exception.NacosException: failed to req API:http://127.0.0.1:8848/nacos/v1/ns/instance. code:400 msg: dom not found: DEFAULT_GROUP@@providers:com.sx.api.OrderAPI:DEFAULT_GROUP

### ISSUE THREAD

**Issue by moayi** [alibaba/nacos#2083]
com.alibaba.nacos.api.exception.NacosException: failed to req API:http://127.0.0.1:8848/nacos/v1/ns/instance/beat. code:400 msg: dom name can only have these characters: 0-9a-zA-Z.:_-; current: DEFAULT_GROUP@@providers:com.sx.api.OrderAPI:DEFAULT_GROUP

	at com.alibaba.nacos.client.naming.net.NamingProxy.callServer(NamingProxy.java:340) ~[nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:367) [nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:304) [nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.net.NamingProxy.sendBeat(NamingProxy.java:227) [nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.beat.BeatReactor$BeatTask.run(BeatReactor.java:109) [nacos-client-1.0.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_221]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) [?:1.8.0_221]
	at java.util.concurrent.FutureTask.run(FutureTask.java) [?:1.8.0_221]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_221]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_221]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_221]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_221]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
15:43:11.172 [com.alibaba.nacos.naming.beat.sender] ERROR com.alibaba.nacos.client.naming - [CLIENT-BEAT] failed to send beat: {"ip":"10.140.186.185","metadata":{"side":"provider","methods":"selectAll,selectByPrimaryKey,updateByPrimaryKey,insert,deleteByPrimaryKey","dubbo":"2.0.2","pid":"16104","interface":"com.sx.api.OrderAPI","generic":"false","revision":"1.0-SNAPSHOT","protocol":"dubbo","application":"dubbo-provider","category":"providers","anyhost":"true","bean.name":"com.sx.api.OrderAPI","group":"DEFAULT_GROUP","timestamp":"1574322184910"},"port":20880,"scheduled":true,"serviceName":"DEFAULT_GROUP@@providers:com.sx.api.OrderAPI:DEFAULT_GROUP","weight":1.0}
java.lang.IllegalStateException: failed to req API:/nacos/v1/ns/instance/beat after all servers([127.0.0.1:8848]) tried: failed to req API:http://127.0.0.1:8848/nacos/v1/ns/instance/beat. code:400 msg: dom name can only have these characters: 0-9a-zA-Z.:_-; current: DEFAULT_GROUP@@providers:com.sx.api.OrderAPI:DEFAULT_GROUP

	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:380) ~[nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:304) ~[nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.net.NamingProxy.sendBeat(NamingProxy.java:227) [nacos-client-1.0.0.jar:?]
	at com.alibaba.nacos.client.naming.beat.BeatReactor$BeatTask.run(BeatReactor.java:109) [nacos-client-1.0.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_221]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) [?:1.8.0_221]
	at java.util.concurrent.FutureTask.run(FutureTask.java) [?:1.8.0_221]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_221]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_221]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_221]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_221]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
15:43:11.176 [RMI TCP Connection(3)-127.0.0.1] ERROR com.alibaba.nacos.client.naming - request 127.0.0.1:8848 failed.
com.alibaba.nacos.api.exception.NacosException: failed to req API:http://127.0.0.1:8848/nacos/v1/ns/instance. code:400 msg: dom not found: DEFAULT_GROUP@@providers:com.sx.api.OrderAPI:DEFAULT_GROUP

what is this?

**Comment by moayi** [comment]
<dubbo:application name="dubbo-provider" />

    <dubbo:protocol  id="protocol" name="dubbo"  port="20880" ></dubbo:protocol>
    
    <dubbo:registry  protocol="nacos" address="127.0.0.1:8848" />

    <dubbo:service interface="com.sx.api.OrderAPI"  ref="orderAPI" group="DEFAULT_GROUP"></dubbo:service>

**Comment by bnrmaster** [comment]
I also encountered the same problem, this should be caused by the mismatched server and client,
my nacos discovery client is 2.1.1 but the server version is 0.9.0.
My problem solved after updating the server to 1.1.4

I downloaded the uncorrect version server because the link of server in Quick Start ( https://nacos.io/en-us/docs/quick-start.html) in English version manual is still 0.9.0.

Hope to fix it and explain the version relevance between server and client.
